components/feature/media/src/test/java/mozilla/components/feature/media/fullscreen/MediaSessionFullscreenFeatureTest.kt

```diff
@@ -94,7 +101,7 @@ class MediaSessionFullscreenFeatureTest {
 
     @Test
     fun `GIVEN the currently selected tab plays landscape media WHEN it enters fullscreen THEN set orientation to landscape`() {
-        val activity: Activity = mock()
+        val activity = Robolectric.buildActivity(Activity::class.java).setup().get()
```

Comment:
Not sure why this change is needed.
Verifying that `setRequestedOrientation` is called or not on the mocked activity should be enough.

Reply:
You're right mock and verify is enough. Will update